### ISSUE TITLE
LARGEST RECTANGULAR AREA IN A HISTOGRAM PROBLEM IMPLEMENTATION USING STACK

### ISSUE THREAD

**Issue by spursbyte** [Abhijit2505/Algorithmic-Treasure-Original#39]
Finding the largest rectangular area possible in a given histogram where the largest rectangle can be made of a number of contiguous bars using stack data structure.
<img width="279" alt="histogram1" src="https://user-images.githubusercontent.com/60615807/103775449-11bce880-5054-11eb-8153-dbf40d4584a9.png">

**Comment by chehak123** [comment]
I would like to work on this under SWOC

**Comment by Abhijit2505** [comment]
@spursbyte please get started

**Comment by spursbyte** [comment]
Yes sir our sem exams just got over yesterday..i will start working soon..sorry for the delay
